### DISCUSSION TITLE
Comparing strings in checkblock is unhandy

### DISCUSSION THREAD

**Discussion by ArminKugler** [postmanlabs/postman-flows#277] [Defect]
When wanting to compare a value from a response with a string this is unhandy.
In this example I enter 

`/data/body/status == FINISHED`

it replaces it to 

`/data/body/status == ""FINISHED""` (notice the quotes twice before and after the string)

For sure this does not work because in the end it compares FINISHED to "FINISHED".
To fix this you need to edit the String **manually** after the **automatic** replacement from ""FINISHED"" to "FINISHED" (removing starting and ending quotes).
Please fix the automatic replacement so it only adds quotes before and after the string only once.

**Comment by saswatds** [comment]
Hey @ArminKugler thanks for the feedback. We are going to improve the expression to be more freeform editing experience and solve it.
